src/components/movieDetailsPageContainer/MovieDetailsPageContainer.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
```

Comment:
Not critical. Just FYI
If you use `MovieDetailsPageContainer` only with `MovieDetailsPage`, you can place it here
![image](https://user-images.githubusercontent.com/17515584/133402631-613ad4f4-9184-4ec0-a6ec-35334229f9e4.png)